src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -34,6 +34,7 @@
 		<PackageReference Include="NUnitLite" Version="3.6.1" />
 		<PackageReference Include="log4net" Version="2.0.8" />
 		<PackageReference Include="Serilog" Version="2.0.0" />
+		<PackageReference Include="Serilog.Sinks.TextWriter" Version="2.1.0" />
```

Comment:
Any reason we need 2.1.0 rather than 2.0.0 like we have defined in our last release? We've defined Serilog 2.0.0.